From f8e251e1062ef401f6220e471cbc5c5b62f3a3dd Mon Sep 17 00:00:00 2001
From: Subrata Chatterjee <schatterjee@habana.ai>
Date: Wed, 28 Jul 2021 19:46:25 +0300
Subject: [PATCH] [SW-45031] greco: add dummy HL_TARGET files

Added HL_TARGET specific (asic, onix, pldm) dummy
files. These will be used to move HL_TARGET
specific common files for GRECO.

Change-Id: I288413bca75020b395bc5778c7c7e02469eb655b
Signed-off-by: Subrata Chatterjee <schatterjee@habana.ai>
---

diff --git a/arch/greco/common/Makefile b/arch/greco/common/Makefile
index c693c15..5daa092 100644
--- a/arch/greco/common/Makefile
+++ b/arch/greco/common/Makefile
@@ -1,4 +1,6 @@
 PREBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH)/../common/common.c)
 PREBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH)/../common/reset.c)
+PREBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH)/../common/$(HL_TARGET).c)
 
 PPBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH)/../common/common.c)
+PPBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH)/../common/$(HL_TARGET).c)
diff --git a/arch/greco/common/asic.c b/arch/greco/common/asic.c
new file mode 100644
index 0000000..2b0934c
--- /dev/null
+++ b/arch/greco/common/asic.c
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier:     GPL-2.0+
+/*
+ *
+ * (C) Copyright 2021 HabanaLabs, Ltd.
+ *
+ */
+
+/* TODO: Greco asic(HL_TARGET) specific common functions to be moved here */
diff --git a/arch/greco/common/onix.c b/arch/greco/common/onix.c
new file mode 100644
index 0000000..b428f5a
--- /dev/null
+++ b/arch/greco/common/onix.c
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier:     GPL-2.0+
+/*
+ *
+ * (C) Copyright 2021 HabanaLabs, Ltd.
+ *
+ */
+
+/* TODO: Greco onix(HL_TARGET) specific common functions to be moved here */
diff --git a/arch/greco/common/pldm.c b/arch/greco/common/pldm.c
new file mode 100644
index 0000000..e7fd49f
--- /dev/null
+++ b/arch/greco/common/pldm.c
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier:     GPL-2.0+
+/*
+ *
+ * (C) Copyright 2021 HabanaLabs, Ltd.
+ *
+ */
+
+/* TODO: Greco pldm(HL_TARGET) specific common functions to be moved here */
